.ci/wercker.sh

```diff
@@ -91,7 +91,7 @@ no-error-orekit)
   # no CI is enforced in project, so to make our build stable we should
   # checkout to latest release/development (annotated tag or hash) or sha that have fix we need
   # git checkout $(git describe --abbrev=0 --tags)
-  git checkout "54f5f2eb410ec42e5410ec8e5c415ff9182d3235"
+  git checkout "76760bfbfb847e227490cd5d3662fca087f1a324"
```

Comment:
please add double quotes in places where spellchecker reports violation, e.g. as""db""123 etc.

Reply:
done.

Reply:
I tried using-

- `"76760""bfbfb""847e227490cd5d3662fca""087f1a324"`

- `"76760""bfbfb""847e227490cd5d3662""fca""087f1a324"`

- `"76760""bfbfb847e2""27490cd5d""3662fca""087f1a324"`
But none of them worked, all showed the same error, can you point out where am I going wrong, spellchecker complains about the words-

- `bfbfb`
- `fca`

Reply:
you need to make it like "76760bf""bf""b847e227490cd5d3662f""ca087f1a324"

I remember someone told me that spellchecker will not complain if there is only 2 letters, so you need double quotes to split this letter sequences

Reply:
done.